text/87_interacting_with_stakeholders.md

```diff
@@ -39,19 +39,19 @@ When you're in school, the teachers tend to have more knowledge than you on what
+If Redli has a chance to talk to Guvvy about some problems with the driver's license renewal process, a bad idea would be to start with technical details of regulation 437-A, subsection 4, paragraph C, and its definition of "resident".  Redli should talk about it in terms that Guvvy understands - things that annoy voters and would make them more likely to vote for AntiGuvvy (who is, of course, the opponent in the upcoming gubernatorial election).  Guvvy is going to be more receptive to knowing that the law doesn't handle people who own houses in two different states than going into detail about the legalese.  This is impactful for politicians because annoyed voters are less likely to vote for the incumbent (note to political science majors - I am dramatically oversimplifying to make a point, so please do not send me angry letters about gubernatorial election theory).
 
-Similarly, when discussing the system or problems that you are having with your boss, put it into terms that they can understand.  Remember that they often have a broader view of the system that you do, and things that you consider very important may not be as important to them.  Conversely, they may not be aware of problems that you consider obvious.  Providing an open flow of information can be very helpful in both directions, allowing you as a tester to understand the system more holistically and allowing you to channel your energy and time appropriately, and allowing your supervisors knowledge of the nitty-gritty of the data, defects, and concerns you are finding in your testing as well as concerns.
+Similarly, when discussing the system or problems that you are having with your boss, put it into terms that they can understand.  Remember that they often have a broader view of the system than you do, and things that you consider very important may not be as important to them.  Conversely, they may not be aware of problems that you consider obvious.  Providing an open flow of information can be very helpful in both directions, allowing you as a tester to understand the system more holistically and channel your energy and time appropriately, while giving your supervisors knowledge of the nitty-gritty of the data, defects, and concerns you are finding in your testing.
```

Comment:
The last sentence deserves to be multiple sentences.  I've realized that I tend to write massive sentences in this.  I've been chopping them up as I come across them.  Thanks!